### PR TITLE
Update 03_foundations

### PR DESCRIPTION
**Pull request by emcaulay**
Having difficulty submitting pull requests

### CODE REVIEW

**Comment by ccronje** [comment]
Hi @emcaulay thanks for the changes. Can you explain what difficulties you are having before I merge this?

**Comment by emcaulay** [comment]
I'm still learning the workflow -- I don't think I know exactly how to submit pull requests. @jt14den was sitting next to me during the mozsprint so he was helping me. I think this one is now correct. I think maybe I just wasn't finishing the process.

**Comment by ostephens** [comment]
I think you've submitted the pull request (PR) absolutely fine. The only suggestion I'd have for future PRs is to split out the different changes into their own PR. This PR mixes very minor changes (e.g. addition of a comma in a long paragraph) with some more substantial re-structuring of the content.

When reviewing the PR it is hard to separate out the minor changes that don't really need much discussion, from the more substantial changes that might warrant more thought/debate.

The other problem here (which is nothing to do with you) is that github highlights the lines that have changed, which is great for code (where lines are short) but not so great for text where a "line" is equivalent to a paragraph - which makes small changes in a paragraph hard to spot - but as I say -  that's not your problem.

Overall the major changes look good to me but I have a minor style quibble that I think it is more usual to capitalise letters when noting Control/Command key shortcuts

File: _episodes/03-foundations.md [hunk 1]
```diff
@@ -59,21 +59,21 @@ Switch Applications | Ctrl | Cmd | Tab |
...
+When working with files for automation or computational purposes, it is more important to focus on meaningful transmission of data as opposed to fancy formatting. Whilst using bold, italics, and colouring to signify headings or to make a visual connection between data elements is common practice, these display-orientated annotations are not (easily) machine-readable and hence can neither be queried and searched nor are appropriate for large quantities of information. One rule of thumb is if you can't find it by ctrl+f/cmd-f it isn't machine readable. Preferable are simple notation schemes such as using a double-asterisk or three hashes to represent a data feature: for example, we could use three question marks to indicate something that needs follow up, chosen because `???` can easily be found with a ctrl+f/cmd+f search.
```
Comment (ostephens) [inline _episodes/03-foundations.md]: Looking around online it looks like it is reasonably standard practice to write keyboard shortcuts wit h capitalised letters - e.g. Ctrl+F or CTRL+F and Cmd+F or CMD+F

**Comment by ccronje** [comment]
Thanks @emcaulay and @ostephens 👍 
I'll merge then make the changes to keyboard shortcuts to capitalise them.

**Pull request merged**